### PR TITLE
Feature/mongo sink

### PR DESCRIPTION
**Pull request by JarHMJ**
**What type of PR is this?**
**PR类型是什么？**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind api-change
> /kind bug
> /kind cleanup
> /kind deprecation
> /kind design
> /kind documentation

**What this PR does / why we need it**:
**这个PR解决了什么问题**:
新增mongodb sink

**Which issue(s) this PR fixes**:
**PR相关联issue**:
无
Fixes #

**Does this PR introduce a breaking change?**:
**PR带来的破坏性变更**:  引入go.mongodb.org/mongo-driver v1.4.6

```docs

```

**Test/Final result**:
**测试/最终运行结果**:
image

### CODE REVIEW

**Review by ringtail — COMMENTED** [review]
Too much vendor update. Could you remove Irrelevant dependencies.

**Comment by JarHMJ** [comment]
> Too much vendor update. Could you remove Irrelevant dependencies.

已经移除多余的vendor，只增加了go.mongodb.org/mongo-driver

File: docs/en/mongodb-sink.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+### MongoDB
+
+To use the MongoDB sink add the following flag:
+```
+    --sink=mongo:?<MONGO_URI>
+```
+
```
Comment (ringtail) [inline docs/en/mongodb-sink.md:7]: Could add the version supported for mongodb

File: sinks/mongo/mongo.go [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+}
+
+func (m *mongoSink) ExportEvents(eventBatch *core.EventBatch) {
+	m.Lock()
+	defer m.Unlock()
+
+	for _, event := range eventBatch.Events {
+		point, err := eventToPoint(event)
+		if err != nil {
+			klog.Warningf("Failed to convert event to point: %v", err)
```
Comment (ringtail) [inline sinks/mongo/mongo.go:100]: add continue to loop control

Comment (JarHMJ) [reply sinks/mongo/mongo.go:100]: fixed

File: docs/en/mongodb-sink.md (hunk 1 above)
Comment (JarHMJ) [reply docs/en/mongodb-sink.md:8]: added

**Pull request merged**